src/Javinizer/Private/Scraper.Dmm.ps1

```diff
@@ -400,14 +399,14 @@ function Get-DmmScreenshotUrl {
     }
 
     process {
-        $trailerHtml = $Webrequest.Content -split '\n'
-        $trailerHtml = $trailerHtml | Select-String -Pattern 'https:\/\/cc3001\.dmm\.co\.jp\/litevideo\/freepv' -AllMatches
-
-        foreach ($trailer in $trailerHtml) {
-            $trailer = (($trailer -split '"')[1] -split '"')[0]
-            $trailerUrl += $trailer
+        $iFrameUrl = 'https://www.dmm.co.jp' + ($Webrequest.Content | Select-String -Pattern "onclick.+sampleplay\('([^']+)'\)").Matches.Groups[1].Value
+        try {
+            $trailerPageUrl = ((Invoke-WebRequest -Uri $iFrameUrl -WebSession $session -Verbose:$false).Content | Select-String -Pattern 'src="([^"]+)"').Matches.Groups[1].Value
+            $trailerUrl = ((Invoke-WebRequest -Uri $trailerPageUrl -WebSession $session -Verbose:$false).Content | Select-String -Pattern '\\/\\/cc3001\.dmm\.co\.jp\\/litevideo\\/freepv[^"]+').Matches.Groups[0].Value -replace '\\', ''

```

Comment:
You can remove the `-WebSession` parameter as $session isn't being assigned in the function.
Seems to be working properly for `dmmja`, but not for `dmm`. 

$iFrameUrl is correct, but $trailerPageUrl seems to be being set with an invalid URL, (i.e. for IPX-219: https://www.dmm.co.jp/en/service/digitalapi/-/html5_player/=/cid=ipx219/mtype=AhRVShI_/service=mono/floor=dvd/mode=/ )

Let's add a string replacement to $trailerPageUrl of `-Replace '/en', ''` and it should output the correct URL.